ios/swift/v2/AmpliSwiftSampleApp/Shared/Plugins/LoggingPlugin.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import AmplitudeSwift
+
+class LoggingPlugin: Plugin {
+    let type: PluginType
+    var amplitude: Amplitude?
+    let jsonEncoder: JSONEncoder
+
+    init() {
+        self.type = .before
+        self.jsonEncoder = JSONEncoder()
+        self.jsonEncoder.outputFormatting = [.prettyPrinted]
+    }
+
+    func setup(amplitude: Amplitude) {
+        self.amplitude = amplitude
+    }
+
+    func execute(event: BaseEvent?) -> BaseEvent? {
+        let jsonData = try? jsonEncoder.encode(event)
+        let json = jsonData != nil ? String(data: jsonData!, encoding: String.Encoding.utf8) : ""
+        print(String(format:"[ampli] event=\(json ?? "")"))
```

Comment:
Not seeing this in the logger. May needs latest updates from `0.4.13`

Reply:
I see it
![image](https://github.com/amplitude/ampli-examples/assets/9509929/5881bbad-c0bd-4fa3-b8c5-9641a3246a29)

Reply:
Working now. Thanks